I think this PR is the reason for this failure

```
------- Stdout: -------
=== RUN   TestLint/TestGolint
=== PAUSE TestLint/TestGolint
=== CONT  TestLint/TestGolint
    --- FAIL: TestLint/TestGolint (52.56s)
        lint_test.go:1408: 
            pkg/cmd/roachtest/sqlalchemy.go:25:5: var supportedSqlAlchemyTag should be supportedSQLAlchemyTag
```

bors r-

--- a/pkg/cmd/roachtest/django.go
+++ b/pkg/cmd/roachtest/django.go
@@ -19,6 +19,9 @@ import (
 var djangoReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
 var djangoCockroachDBReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)$`)
 
+var djangoSupportedTag = "3.0.6"
+var djangoCockroachDBSupportedTag = "3.0"
+
 func registerDjango(r *testRegistry) {
 	runDjango := func(
 		ctx context.Context,
@@ -103,6 +106,7 @@ func registerDjango(r *testRegistry) {
 			t.Fatal(err)
 		}
 		c.l.Printf("Latest Django release is %s.", djangoLatestTag)
+		c.l.Printf("Supported Django release is %s.", djangoSupportedTag)
 
 		if err := repeatGitCloneE(
 			ctx,
@@ -110,7 +114,7 @@ func registerDjango(r *testRegistry) {
 			c,
 			"https://github.com/django/django/",
 			"/mnt/data1/django",
-			djangoLatestTag,
+			djangoSupportedTag,
 			node,
 		); err != nil {
 			t.Fatal(err)
@@ -123,6 +127,7 @@ func registerDjango(r *testRegistry) {
 			t.Fatal(err)
 		}
 		c.l.Printf("Latest django-cockroachdb release is %s.", djangoCockroachDBLatestTag)
+		c.l.Printf("Supported django-cockroachdb release is %s.", djangoCockroachDBSupportedTag)
 
 		if err := repeatGitCloneE(
 			ctx,
@@ -130,7 +135,7 @@ func registerDjango(r *testRegistry) {
 			c,
 			"https://github.com/cockroachdb/django-cockroachdb",
 			"/mnt/data1/django/tests/django-cockroachdb",
-			djangoCockroachDBLatestTag,
+			djangoCockroachDBSupportedTag,
 			node,
 		); err != nil {
 			t.Fatal(err)

--- a/pkg/cmd/roachtest/sqlalchemy.go
+++ b/pkg/cmd/roachtest/sqlalchemy.go
@@ -22,6 +22,8 @@ import (
 var sqlAlchemyResultRegex = regexp.MustCompile(`^(?P<test>test.*::.*::[^ \[\]]*(?:\[.*])?) (?P<result>.*)$`)
 var sqlAlchemyReleaseTagRegex = regexp.MustCompile(`^rel_(?P<major>\d+)_(?P<minor>\d+)_(?P<point>\d+)$`)
 
+var supportedSQLAlchemyTag = "rel_1_3_17"
+
 // This test runs the SQLAlchemy dialect test suite against a single Cockroach
 // node.
 
@@ -54,9 +56,7 @@ func registerSQLAlchemy(r *testRegistry) {
 			t.Fatal(err)
 		}
 		c.l.Printf("Latest sqlalchemy release is %s.", latestTag)
-
-		// TODO: using the latest version requires the dialect to implement `get_isolation_levels`
-		latestTag = "rel_1_3_14"
+		c.l.Printf("Supported sqlalchemy release is %s.", supportedSQLAlchemyTag)
 
 		if err := repeatRunE(
 			ctx, c, node, "update apt-get",
@@ -142,7 +142,7 @@ func registerSQLAlchemy(r *testRegistry) {
 			c,
 			"https://github.com/sqlalchemy/sqlalchemy.git",
 			"/mnt/data1/sqlalchemy",
-			latestTag,
+			supportedSQLAlchemyTag,
 			node,
 		); err != nil {
 			t.Fatal(err)

--- a/pkg/cmd/roachtest/typeorm.go
+++ b/pkg/cmd/roachtest/typeorm.go
@@ -18,6 +18,7 @@ import (
 )
 
 var typeORMReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
+var supportedTypeORMRelease = "0.2.24"
 
 // This test runs TypeORM's full test suite against a single cockroach node.
 func registerTypeORM(r *testRegistry) {
@@ -40,6 +41,7 @@ func registerTypeORM(r *testRegistry) {
 			t.Fatal(err)
 		}
 		c.l.Printf("Latest TypeORM release is %s.", latestTag)
+		c.l.Printf("Supported TypeORM release is %s.", supportedTypeORMRelease)
 
 		if err := repeatRunE(
 			ctx, c, node, "update apt-get", `sudo apt-get -qq update`,
@@ -92,7 +94,7 @@ func registerTypeORM(r *testRegistry) {
 			c,
 			"https://github.com/typeorm/typeorm.git",
 			"/mnt/data1/typeorm",
-			latestTag,
+			supportedTypeORMRelease,
 			node,
 		); err != nil {
 			t.Fatal(err)

